dali/base/dadfs.cpp

```diff
@@ -5116,16 +5119,19 @@ class CDistributedSuperFile: public CDistributedFileBase<IDistributedSuperFile>
                 if (!orderedSubFiles[i])
                     ThrowStringException(-1, "CDistributedSuperFile: SuperFile %s: missing subfile part number %d of %d", logicalName.get(), i+1, n);
             }
+            bool superfileRestricted = root->getPropBool("@restricted", false);
```

Comment:
Needs changing, inlight of other PR-12940. But should perhaps either call isRestrictedAccess() for that reason, or an inline version of it.

Reply:
I have already defined isRestrictedAccess to return true if subfile is restricted or superfile is restricted.  I'll modify inlight of PR-12940